src/workflows-app/components/WorkflowInfoBox.test.ts

```diff
@@ -0,0 +1,92 @@
+import { act, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { getLink } from 'src/libs/nav';
+import { makeCompleteDate } from 'src/libs/utils';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import { metadata as mockRunDetailsMetadata } from 'src/workflows-app/fixtures/test-workflow';
+
+import { WorkflowInfoBox, WorkflowInfoBoxProps } from './WorkflowInfoBox';
+
+const mockWorkflowInfoBoxProps: WorkflowInfoBoxProps = {
+  name: 'mock-workspace-name',
+  namespace: 'mock-workspace-namespace',
+  submissionId: 'mock-submission-uuid',
+  workflowId: 'mock-workflow-uuid',
+  workspaceId: 'mock-workspace-uuid',
+  showLogModal: jest.fn(),
+};
+
+jest.mock('src/libs/nav', () => ({
+  getCurrentUrl: jest.fn().mockReturnValue(new URL('https://app.terra.bio')),
+  getLink: jest.fn(),
+  goToPath: jest.fn(),
+}));
+
+jest.mock('src/workflows-app/utils/cromwell-metadata-utils', () => ({
+  ...jest.requireActual('src/workflows-app/utils/cromwell-metadata-utils'),
+  fetchMetadata: jest.fn().mockReturnValue(mockRunDetailsMetadata),
+}));
+
+describe('Workflow Info Box Rendering', () => {
+  it('should render the workflow timing section', async () => {
+    await act(() => render(h(WorkflowInfoBox, mockWorkflowInfoBoxProps)));
```

Comment:
`render` is synchronous and does not return a promise. When called with a synchronous function, `act` is also synchronous. Thus, there's no need for `await` here.

```suggestion
    act(() => render(h(WorkflowInfoBox, mockWorkflowInfoBoxProps)));
```

Reply:
Nick and I discussed this offline - `await` is fine to stay here since the WorkflowInfoBox is making an async call to load the data needed to render.